pipeline_steps/multi_node_aio_prepare.groovy

```diff
@@ -34,7 +34,7 @@ def prepare() {
           # RAM to VMs -- since we have RAM to spare we double that assigned to
           # infra nodes.
           echo "infra_vm_server_ram: 16384" | sudo tee -a playbooks/group_vars/all.yml
-          cp -a ${WORKSPACE}/rpc-gating/scripts/dynamic_inventory.py playbooks/inventory
+          ${WORKSPACE}/rpc-gating/scripts/dynamic_inventory.py playbooks/inventory/rpc-inventory.yaml
```

Comment:
It looks like the ``dynamic_inventory.py`` script is an inventory generator, rather than a dynamic inventory. In other words - it is used to generate a static inventory in ``playbooks/inventory/rpc-inventory.yaml``. If so, perhaps the name should change to something more appropriate, like ``mnaio-inventory-generate.py`` so that where it's used is clear?

Reply:
I could rename it. I didn't write it I just updated what we had however I'd be happy to rename it if you think it best.

Reply:
done.